extensions/ql-vscode/src/view/common/SuggestBox/useEffectEvent.ts

```diff
@@ -9,6 +9,7 @@ import { useCallback, useInsertionEffect, useRef } from "react";
  *
  * @param callback The callback to call when the event is triggered.
  */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
We could rewrite this function to avoid `any` but I'm hesitant because I don't fully understand what it's doing and it's copied verbatim from another source. We could alternatively rewrite this.